api/Dockerfile

```diff
@@ -1,23 +1,24 @@
-FROM abihf/wget:latest AS Downloader
+FROM abihf/wget:latest AS kubeseal-downloader
 RUN wget https://github.com/bitnami-labs/sealed-secrets/releases/download/v0.11.0/kubeseal-linux-amd64 && \
     mv kubeseal-linux-amd64 /tmp/kubeseal
 
 FROM python:3.8-alpine3.12
-RUN mkdir /kubeseal-webgui
-COPY --from=Downloader /tmp/kubeseal /kubeseal-webgui
-COPY app/ /kubeseal-webgui/
-COPY requirements.txt /kubeseal-webgui/
 
-WORKDIR /kubeseal-webgui
+ENV PORT 5000
```

Comment:
@Jaydee94 Intentionally didn't use args. Depending on the environment where the images is run, it might be necessary to change the port before runtime (e.g. if 5000 is already in use).